qiita_db/metadata_template/test/test_base_metadata_template.py

```diff
@@ -67,6 +67,33 @@ def test_clean_validate_template(self):
         with self.assertRaises(IncompetentQiitaDeveloperError):
             MT._clean_validate_template(None, 1)
 
+    def test_identify_pgsql_reserved_words(self):
+        MT = qdb.metadata_template.base_metadata_template.MetadataTemplate
+        results = MT._identify_pgsql_reserved_words_in_column_names([
+            'select',
+            'column',
+            'just_fine1'])
+        self.assertTrue(set(results) == {'column', 'select'})
```

Comment:
sent via PM but also to have it here, this can be: self.assertItemsEqual(a, b)

Reply:
Done!